<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDy"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editDy">
       <property name="text">
        <string>dy</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="editDz">
       <property name="text">
        <string>dz</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDz"/>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="lineMove">
       <property name="text">
        <string>Перенос</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxDx"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="editDx">
       <property name="text">
        <string>dx</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="move">
       <property name="text">
        <string>Перенести</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_8">
       <property name="text">
        <string>ky</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="scale">
       <property name="text">
        <string>Масштабировать</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxKz"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_9">
       <property name="text">
        <string>kz</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="text">
        <string>Масштабирование</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxKx"/>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxKy"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="text">
        <string>kx</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>191</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_10">
       <property name="text">
        <string>angle x</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>Поворот</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAz"/>
     </item>
     <item row="2" column="0">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAx"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="text">
        <string>angle z</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="text">
        <string>angle y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxAy"/>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QPushButton" name="rotate">
       <property name="text">
        <string>Повернуть</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_2">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Файл</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
